[DOCS] Adds scope to monitoring

lcawl opened the pull request:
Fixes https://github.com/elastic/elasticsearch/issues/57217

Related to https://github.com/elastic/beats/pull/18547

This PR also switches the instructions in https://www.elastic.co/guide/en/elasticsearch/reference/master/configuring-metricbeat.html so that it matches https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-module-elasticsearch.html
and recommends the Elasticsearch module instead of the Elasticsearch X-Pack module. 

## Preview
https://elasticsearch_57852.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/configuring-metricbeat.html

lcawl committed:
[DOCS] Adds scope to monitoring

elasticmachine commented:
Pinging @elastic/es-core-features (:Core/Features/Monitoring)

elasticmachine commented:
Pinging @elastic/es-docs (>docs)

ycombinator commented:
Thanks for the PR. I think we should keep the original step 3 as is, the one in which we ask users to enable the Elasticsearch X-Pack module. 

Also, we can just remove step 4 in `master` and `7.x`, as enabling the Elasticsearch X-Pack module now auto-enables the appropriate metricsets (starting from 7.8.0; see https://github.com/elastic/beats/pull/17609).

lcawl committed:
[DOCS] Reverts to using elasticsearch x-pack module

lcawl committed:
[DOCS] Adds callouts for example elasticsearch-xpack.yml

lcawl commented:
> I think we should keep the original step 3 as is, the one in which we ask users to enable the Elasticsearch X-Pack module.

Thanks for the feedback! I've reverted to the use of the Elasticsearch X-Pack module but have added a link for the alternative method.

> Also, we can just remove step 4

I've left it in to show the placement of the scope attribute, but have removed the metricsets to align with what I see in my 7.8 tests.  I've also added callouts to the example so it's a bit clearer.

ycombinator reviewed (approved):
LGTM. Sorry for the delay!
